staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -108,6 +108,13 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		userInfo, _ := request.UserFrom(ctx)
 		staticAdmissionAttributes := admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo)
 		admissionCheck := func(updatedObject runtime.Object, currentObject runtime.Object) error {
+			isNotZeroObject, err := hasUID(currentObject)
+			if err != nil {
+				return fmt.Errorf("unexpected error when extracting UID from oldObj: %v", err.Error())
+			} else if !isNotZeroObject {
+				// if we allow this some day, we have this TODO: call the mutating admission chain with the CREATE verb instead of UPDATE
+				panic("a PATCH cannot create objects")
+			}
```

Comment:
all this is kind of ugly, agreeing with @jennybuckley. Let's condense the change for the patch here into the simple TODO, no panic. Lgtm then.

Reply:
@sttts Done